### ISSUE TITLE
Clarify doc about how to mount NFS shared volumes

### ISSUE THREAD

**Issue by stuclem** [vmware/vic-product#2410]
Per email thread with @dbarkelew @tschwaller and others, there is some confusion about how to mount an NFS share:

-----

it is confusing why we have two ways documented for providing an NFS Volume Store in VIC.
 
1)	[Mount an NFS Share Point in the VCH Endpoint VM](https://vmware.github.io/vic-product/assets/files/html/1.5/vic_vsphere_admin/vch_mount_nfsshare.html) Is this option maybe just for testing purposes? 
” If you deploy VCHs that use NFS share points as volume stores, you can test the connections to these share points by mounting the NFS server from within the VCH endpoint VM. Confirming that you can mount the NFS share point from within the endpoint VM confirms that the NFS volume store configuration will also work.”
2) [Specify Volume Datastores](https://vmware.github.io/vic-product/assets/files/html/1.5/vic_vsphere_admin/volume_stores.html#nfsoptions):  --volume-store nfs://nfs_server/path_to_share_point:default 
 

-----

**Comment by stuclem** [comment]
Doc is currently misleading. Setting to severity 1.

**Comment by stuclem** [comment]
@vidya-v I believe that @lgayatri worked on the NFS implementation. Can you please look into this one with her and @jitinkumar2018 ?

**Comment by jitinkumar2018** [comment]
As of now, we have only tested specifying the volume store during VCH creation in the following manner:
--volume-store CPBU_2TB_02/vchvolumes/:backed-up-encrypted , 
where CPBU_2TB_02 is the name of the NFS datastore that is mounted on all the ESXi hosts in the cluster. 

We have not tried mounting the NFS Volume Store using : 
" --volume-store nfs://nfs_server/path_to_share_point:default"

In order to verify whether we can mount the NFS share point from inside the VCH vm, the NFS datastore should allow anonymous mounting which is right now not configured that way.

**Comment by jitinkumar2018** [comment]
Hi @stuclem @vidya-v,
I verified the two steps mentioned in the docs about 

1) Mount an NFS Share Point in the VCH Endpoint VM 
2)Specify Volume Datastores

The First step is used to check if the VCH vm (NFS client) can use/communicate with the NFS server. 
If this step is successful, then the VCH can be configured to create a volume store backed up by this NFS server by specifying the NFS share point in the --volume-store parameter

In the VCH create command using 'vic-machine-linux' CLI, I added :
--volume-store nfs://<nfs-server-ip>/home/vmlib/nfsroot:default
( /home/vmlib/nfsroot is the share point I created and exposed on the nfs server )

We can then create docker volumes on this VCH by running:
docker  -H <VCH-IP>:2376 volume create --name test --opt VolumeStore=default

and use these docker volumes while creating containers. 
Eg.
docker -H <VCH-IP>:2376 --tls run -itd -v test:/mnt/test alpine

Hence Step 1) is just a test
and Step 2) shows us how to create a volume store backed up by NFS
